Dart/build.log

```diff
@@ -1,5 +1,5 @@
-Built on 2025-01-28 15:34:10+01:00
+Built on 2025-03-06 09:52:17+01:00
 Cloning into 'wasm_gc_benchmarks'...
-cf32ca4 Recompile all benchmarks
+f80892d Update Dart SDK, switch wasm to -O2, recompile performance benchmarks
```

Comment:
Did you manually edit this? This is an output file from `build.sh` normally so next time someone updates they'll see the old log again.

Is there a build with the bits you're uploading here saved somewhere? Maybe we can update that script to pull from there?

Reply:
This particular line is the abbreviated commit hash and commit message. See hash & commit message from https://github.com/mkustermann/wasm_gc_benchmarks/commit/f80892dcc9578cb864ebb870f7628c8c825fe890 

That being said I removed one line from the generated `build log`: `git clone` outputed something that's to be interpreted by a terminal rather than normal text (`git clone` shows this `Updating ... (x/y)` where it constantly updates the x/y information of the same line). It wasn't there in the old `build.log` and isn't normal text, so I removed that line